doc-surrealdb_versioned_docs/version-latest/querying/index.mdx

```diff
@@ -0,0 +1,182 @@
+
+- **CLI**: The SurrealDB Command Line Interface (CLI) provides a powerful way to [interact with the database directly from your terminal](/docs/surrealdb/cli). You can execute SurrealQL queries, manage database structures, and perform administrative tasks using the CLI.
+
+- **WebSocket**: SurrealDB supports WebSocket connections, allowing you to execute SurrealQL queries in real-time. This option is ideal for applications that require low-latency communication and real-time updates.
+
+- **HTTP**: You can send HTTP requests to the [`/sql` endpoint](/docs/surrealdb/integration/http#sql) to execute SurrealQL queries. This method is useful for integrating SurrealDB with web applications and services that communicate over HTTP.
+  - **Postman**: Using Postman or any other HTTP client, you can send a `POST` request to the `/sql` endpoint with your SurrealQL query in the body. This method provides flexibility and can be useful for testing and automation purposes.
+
+- **RPC**: SurrealDB also supports RPC, allowing you to interact with the database programmatically over a network.
+
+#### Using RPC
+
+SurrealDB also supports RPC, allowing you to interact with the database programmatically over a network.
+
+1. **Set up an RPC client**: Depending on your programming language, you can use various libraries to create an RPC client. Here is an example using JavaScript with the `node-fetch` library:
+
+   ```javascript
+   const fetch = require('node-fetch');
+
+   async function querySurrealDB() {
+     // Create a new person record
+     await fetch('http://localhost:8000/rpc', {
+       method: 'POST',
+       headers: {
+         'Content-Type': 'application/json',
+       },
+       body: JSON.stringify({
+         method: 'query',
+         params: ['CREATE person SET name = "John Doe"; SELECT * FROM person;'],
+       }),
+     });
+
+     const data = await response.json();
+     console.log(data);
+   }
+
+   querySurrealDB();
+   ```
+
+2. **Execute the RPC call**: Run your script to send the RPC request and receive the response from SurrealDB.
+
+
+<ThemedImage
+  alt="QL"
+  width="70"
+  style={{ float: 'left', marginRight: '10px' }}
+  sources={{
+    light: useBaseUrl('img/light/gql.png'),
+    dark: useBaseUrl('img/gql-icon.png'),
+  }}
+/>
+
+## GraphQL
+
+<Since v="v2.0.0" />
+
+SurrealDB also supports GraphQL, allowing you to interact with the database using the familiar syntax. Currently, you can use the [GraphQL integration via Surrealist](/docs/surrealdb/querying/graphql/surrealist), our intuitive user interface specifically designed for SurrealDB, or [over HTTP](/docs/surrealdb/querying/graphql/http) via a GraphQL Client such as GraphiQL or [Postman](/docs/surrealdb/querying/graphql/http#postman) at the `localhost:8000/graphql` endpoint.
```

Comment:
Typo: GraphiQL

Also small nit: the sentence starts with "you can use" with one option and then goes on to say "over HTTP" which together makes something like "You can use GraphQL or over HTTP". Maybe change to "access the database over HTTP..."